### DISCUSSION TITLE
Are there any ways to seed data after schema migration included newly created tables/columns at production database through prisma?

### DISCUSSION THREAD

**Discussion by Exslims** [prisma/prisma#17350] [Q&A] [answered]
Hi anyone! 
Maybe it is a stupid question but sometimes, after migration schema you have to seed new data to newly created tables/columns in production database. 
So are there any ways to do it through prisma (by js code) instead of doing to through plain SQL?

**Comment by nurul3101** [comment, marked as answer]
Hello @Exslims 👋 

If you want to add seed new data once migrations are completed, then you can create a migration with `--create-only` flag ([Reference](https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/customizing-migrations)) and add the SQL for seed data, this solution is using plain SQL.

Now, for doing it through JS code, I usually follow the [Quickstart](https://www.prisma.io/docs/getting-started/quickstart) guide to create a minimal application and point it to the production database and write the queries in `script.ts` file. Maybe this is something that could work for you as well.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
